packages/lesswrong/lib/collections/automatedContentEvaluations/newSchema.ts

```diff
@@ -34,12 +34,44 @@ const schema = {
       typescriptType: "{ sentence: string; score: number; }[]",
     },
     graphql: {
-      outputType: "[SentenceScore]",
-      canRead: ["admins"],
+      outputType: "[SentenceScore]!",
+      canRead: ["guests"],
     },
+  },
 
+  aiChoice: {
+    database: {
+      type: "VARCHAR(255)",
+      nullable: false,
+      typescriptType: '"accept" | "review"',
```

Comment:
This should be done via a `validation: { allowedValues: ['accept', 'review'] }` field in the `graphql` section (which will cause the correct type to be codegen'd).